app/models/shelter.rb

```diff
@@ -2,10 +2,29 @@ class Shelter < ApplicationRecord
   has_many :pets, dependent: :destroy
   has_many :reviews, dependent: :destroy
 
+  validates_presence_of :name
+  validates_presence_of :address
+  validates_presence_of :city
+  validates_presence_of :state
+  validates_presence_of :zip
```

Comment:
Had to validate presence so the fields were not optional

Reply:
Can we add a validation test for our shelter model?